### ISSUE TITLE
Exporter panics trying to convert prometheus.GaugeVec to main.settableCounterVec.

### ISSUE THREAD

**Issue by jomach** [mesos/mesos_exporter#87]
I was trying this exporter out and I get this error message. 
```
panic: interface conversion: prometheus.Collector is *prometheus.GaugeVec, not *main.settableCounterVec

goroutine 12 [running]:
main.newMasterCollector.func20(0xc4202c4120, 0x8abfe0, 0xc42000e080, 0x0, 0x0)
	/go/src/github.com/mesos/mesos_exporter/master.go:419 +0x6f7
main.(*metricCollector).Collect(0xc42009eb30, 0xc4200ba420)
	/go/src/github.com/mesos/mesos_exporter/common.go:295 +0x1e7
github.com/mesos/mesos_exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func2(0xc420024470, 0xc4200ba420, 0x8ac1e0, 0xc42009eb30)
	/go/src/github.com/mesos/mesos_exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:383 +0x61
created by github.com/mesos/mesos_exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather
	/go/src/github.com/mesos/mesos_exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:381 +0x302
```

Mesos version: 1.3.2
Prometheus version: 2.1.0
mesos_exporter: Running on suse 12 sp2
Mesos runs over https and I ran: ./mesos_exporter -master https://serverX:5050

**Comment by jpeach** [comment]
I think this is a regression from #82. I believe @jacobjanco has a fix.

**Comment by jacobjanco** [comment]
I'll get this cleared internally and submit the fix. Thanks @jomach

**Comment by jacobjanco** [comment]
https://github.com/mesos/mesos_exporter/pull/91

**Comment by jpeach** [comment]
Fixed by #86
